packages/server/assets/workspacesCore/typedefs/workspaces.graphql

```diff
@@ -1,3 +1,72 @@
+extend type Query {
+  workspace(id: String!): Workspace!
+}
+
+input WorkspaceCreateInput {
+  name: String!
+  description: String
+  logoUrl: String
+}
+
+input WorkspaceUpdateInput {
+  id: String!
+  name: String
+  description: String
+  logoUrl: String
+}
+
+input WorkspaceRoleUpdateInput {
+  userId: String!
+  workspaceId: String!
+  role: WorkspaceRole!
+}
+
+input WorkspaceRoleDeleteInput {
+  userId: String!
+  workspaceId: String!
+}
+
+extend type Mutation {
+  workspaceMutations: WorkspaceMutations!
+}
+
+type WorkspaceMutations {
+  create(input: WorkspaceCreateInput!): Workspace!
+  delete(workspaceId: String!): Workspace!
+  update(input: WorkspaceUpdateInput!): Workspace!
+  updateRole(input: WorkspaceRoleUpdateInput!): Boolean!
+  deleteRole(input: WorkspaceRoleDeleteInput!): Boolean!
+  invites: WorkspaceInviteMutations!
+}
+
+input WorkspaceInviteCreateInput {
+  """
+  Either this or userId must be filled
+  """
+  email: String
+  """
+  Either this or email must be filled
+  """
+  userId: String
+  """
+  Defaults to the member role, if not specified
+  """
+  role: WorkspaceRole
+}
+
+input WorkspaceInviteUseInput {
+  workspaceId: String!
+  token: String!
+  accept: Boolean!
+}
+
+type WorkspaceInviteMutations {
+  create(workspaceId: String!, input: WorkspaceInviteCreateInput!): Workspace!
```

Comment:
Most of these (except use()`) probably need a `@hasScope` check for invites scopes